packages/fx-core/src/common/local/index.ts

```diff
@@ -5,3 +5,4 @@
 export * from "./localEnvManager";
 export * from "./projectSettingsHelper";
 export * from "./taskDefinition";
+export * from "./constants";
```

Comment:
Is this for `FolderName`?
`vscode-extension/src/debug/constants.ts` also has a few folder names, considering dedup them in future changes.

Reply:
Updated in PR https://github.com/OfficeDev/TeamsFx/pull/3629